src_c/IMB_ones_accu.c

```diff
@@ -150,6 +150,10 @@ Output variables:
     if (c_info->rank < 0)
         *time = 0.;
     else {
+
```

Comment:
Are we losing the synchronization step in aggregate?

Reply:
Hi @juhlee-microsoft , sure - those barriers works not only for nonaggregate mode, all processes do barriers before starting aggregate\nonaggregate benchmark.
We have two independent "if" on line 157 and 185.

Reply:
Acked. Thanks for letting me know.

Reply:
Hi @VinnitskiV  I do not understand why these Barriers only apply to non_aggregation mode. For example for MPI_Get tests both modes have these Barriers: 
https://github.com/intel/mpi-benchmarks/blob/master/src_c/IMB_ones_unidir.c#L297
https://github.com/intel/mpi-benchmarks/blob/master/src_c/IMB_ones_unidir.c#L384